{
	"schema_version": "1.6.0",
	"id": "MAL-2025-936",
	"modified": "2025-02-03T17:07:29Z",
	"published": "2025-02-03T17:07:29Z",
	"summary": "Malicious code in intgraphx (PyPI)",
	"details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "intgraphx",
				"purl": "pkg:pypi/intgraphx"
			},
			"versions": [
				"0.1"
			],
			"database_specific": {
				"source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/intgraphx/MAL-2025-936.json"
			}
		}
	],
	"credits": [
		{
			"name": "ReversingLabs",
			"contact": [
				"https://www.reversinglabs.com"
			],
			"type": "FINDER"
		}
	],
	"database_specific": {
		"malicious-packages-origins": [
			{
				"id": "RLMA-2025-00476",
				"import_time": "2025-02-03T18:38:06.925488687Z",
				"modified_time": "2025-02-03T17:07:29Z",
				"sha256": "1d0358aaf573ea9d5800068018a195abc017e030fb094e87b2a4c526db595eb8",
				"source": "reversing-labs",
				"versions": [
					"0.1"
				]
			}
		]
	}
}
